rustls/src/msgs/codec.rs

```diff
@@ -270,9 +270,9 @@ impl<'a> LengthPrefixedBuffer<'a> {
     pub(crate) fn new(size_len: ListLength, buf: &'a mut Vec<u8>) -> LengthPrefixedBuffer<'a> {
         let len_offset = buf.len();
         buf.extend(match size_len {
-            ListLength::U8 => &[0][..],
-            ListLength::U16 => &[0, 0],
-            ListLength::U24 { .. } => &[0, 0, 0],
+            ListLength::U8 => &[0xff][..],
```

Comment:
Commit message has some interesting spellings of "interpreted" like "intepretted" and "interpretted".

Reply:
shameful 🙈